test/modules/standard/IO/saru/getParentName/parentName.chpl

```diff
@@ -0,0 +1,10 @@
+use IO;
+
+/* Tests IO.file.getParentName() function */
+proc main() {
+  var f = open('parentNameTest.txt', iomode.r),
+      c = f.reader(),
+      err: syserr;
+
+  writeln(f.getParentName(err));
```

Comment:
This is a good first start for testing, but ideally we would have a couple more.  What other cases should you be sure to cover?